selftests/functional/runner_pip.py

```diff
@@ -0,0 +1,61 @@
+import os
+import sys
+import unittest
+
+from avocado.utils import process
+from selftests.utils import BASEDIR
+
+RUNNER = f"{sys.executable} -m avocado.plugins.runners.pip"
+
+
+class RunnableRun(unittest.TestCase):
+    def test_no_kwargs(self):
+        res = process.run(f"{RUNNER} runnable-run -k pip", ignore_status=True)
+        self.assertIn(b"'status': 'started'", res.stdout)
+        self.assertIn(b"'status': 'finished'", res.stdout)
+        self.assertIn(b"'time': ", res.stdout)
+        self.assertEqual(res.exit_status, 0)
+
+    @unittest.skipUnless(
```

Comment:
An idea: I think we're at a point where we need to stop depending on the `CI` environment flag, and determine whether the environment the test is running:
* can be written permanently and destructively
* has network access

Of course we can look into `CI` environment variables and the like, but it would make the `SKIPs` much more manageable.